src/main/resources/interface-specification.yml

```diff
@@ -70,6 +70,12 @@ paths:
             minimum: 1
             maximum: 500
             default: 100
+        - name: fromAllOrganizations
```

Comment:
I agree with what you were saying about this flag, probably splitting this into 2 different endpoints would be better.
I don't expect a client to know how the service works under the hood and that a different partition should be used in each case